src/plugins/intel_cpu/src/transformations/cpu_opset/x64/pass/convert_precision_i64_i32.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/pass.hpp"
+
+namespace ov {
+namespace intel_cpu {
+class ConvertPrecisionI64ToI32: public ov::pass::ModelPass {
```

Comment:
Can you please add a short pass description? I think it worth to elaborate here that we insert I64->I32 converts on inputs and outputs for the operations which don't support I64 precision natively

Reply:
Done